File: abc/abc153/a.cpp
// https://atcoder.jp/contests/abc153/submissions/26232305

#include <bits/stdc++.h>
using namespace std;
int main() {
  int h, a, count = 0;
  cin >> h >> a;
  while(h > 0) {
    h -= a;
    count++;
  }
  cout << count;
}

File: abc/abc153/a.java
// https://atcoder.jp/contests/abc153/submissions/41638938

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int h = stdIn.nextInt();
    int a = stdIn.nextInt();

    System.out.println((h + a - 1) / a);
  }
}
